gst/tensor_decoder/tensordec.c

```diff
@@ -740,11 +740,12 @@ gst_tensordec_label_set_output (GstTensorDec * self, GstBuffer * outbuf,
   GstMemory *out_mem;
   GstMapInfo out_info;
 
+  gst_buffer_set_size (outbuf, 0);
   g_assert (gst_buffer_get_size (outbuf) == 0);
 
   len = strlen (label);
 
-  out_mem = gst_allocator_alloc (NULL, (len + 1), NULL);
+  out_mem = gst_allocator_alloc (NULL, len, NULL);
```

Comment:
please remove line 753 if changes mem size. 
((char *) out_info.data)[len] = '\0';

Reply:
OK I will change this as you recommand